src/N98/Magento/Command/Developer/EmailTemplate/UsageCommand.php

```diff
@@ -61,32 +60,34 @@ protected function findEmailTemplates()
              * is not available, this is a workaround for that
              */
             if (!method_exists($template, 'getSystemConfigPathsWhereUsedCurrently')){
-                $instance = new \Mage_Adminhtml_Model_Email_Template();
+                $instance = new Mage_Adminhtml_Model_Email_Template();
                 $template = $instance->load($template->getId());
             }
 
             $configPaths = $template->getSystemConfigPathsWhereUsedCurrently();
 
-            if (count($configPaths)){
-                foreach ($configPaths as $configPath) {
-                    $this->infos[] = array(
-                        'id'            => $this->sanitizeEmailProperty($template->getId()),
-                        'Template Code' => $this->sanitizeEmailProperty($template->getTemplateCode()),
-                        'Scope'         => $this->sanitizeEmailProperty($configPath['scope']),
-                        'Scope Id'      => $this->sanitizeEmailProperty($configPath['scope_id']),
-                        'Path'          => $this->sanitizeEmailProperty($configPath['path']),
-                    );
-                }
-            } else {
-                $this->infos[] = array(
+            if (!count($configPaths)){
+                $return[]  = array(
```

Comment:
Sure, can you see this: https://github.com/MikeParkin/n98-magerun/compare/develop...ktomk:patch-1 ? Instead of manipulating $return, it manipulates the $configPaths (for the unused case).